src/ludamus/templates/chronology/event.html

```
@@ -370,7 +370,9 @@ <h2 class="text-xl font-bold text-foreground">{% translate "Sessions" %}</h2>
                 <div id="active-filter-chips"
                      class="filter-chips-bar"
                      data-remove-filter-label="{% translate "Remove filter" %}"
-                     data-clear-all-label="{% translate "Clear all" %}">
+                     data-clear-all-label="{% translate "Clear all" %}"
+                     data-min-age-label="{% translate "Age ≥" %}"
+                     data-max-age-label="{% translate "Age ≤" %}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)